openedx/core/djangoapps/user_authn/serializers.py

```diff
@@ -0,0 +1,77 @@
+"""
+MFE Context API Serializers
+"""
+
+from rest_framework import serializers
+
+
+class ProvidersSerializer(serializers.Serializer):
+    """
+    Providers Serializers
+    """
+
+    id = serializers.CharField(allow_null=True)
+    name = serializers.CharField(allow_null=True)
+    iconClass = serializers.CharField(allow_null=True)
+    iconImage = serializers.CharField(allow_null=True)
+    skipHintedLogin = serializers.BooleanField(default=False)
+    skipRegistrationForm = serializers.BooleanField(default=False)
+    loginUrl = serializers.CharField(allow_null=True)
+    registerUrl = serializers.CharField(allow_null=True)
+
+
+class PipelineUserDetailsSerializer(serializers.Serializer):
+    """
+    Pipeline User Details Serializers
+    """
+
+    username = serializers.CharField(allow_null=True)
+    email = serializers.CharField(allow_null=True)
+    name = serializers.CharField(source='fullname', allow_null=True)
+    firstName = serializers.CharField(source='first_name', allow_null=True)
+    lastName = serializers.CharField(source='last_name', allow_null=True)
+
+
+class ContextDataSerializer(serializers.Serializer):
+    """
+    Context Data Serializers
+    """
+
+    currentProvider = serializers.CharField(allow_null=True)
+    platformName = serializers.CharField(allow_null=True)
+    providers = serializers.ListField(
+        child=ProvidersSerializer(),
+        allow_null=True
+    )
+    secondaryProviders = serializers.ListField(
+        child=ProvidersSerializer(),
+        allow_null=True
+    )
+    finishAuthUrl = serializers.CharField(allow_null=True)
+    errorMessage = serializers.CharField(allow_null=True)
+    registerFormSubmitButtonText = serializers.CharField(allow_null=True)
+    autoSubmitRegForm = serializers.BooleanField(default=False)
```

Comment:
This is behind a check. ([reference](https://github.com/openedx/edx-platform/blob/9caa31bb063e23ab6a59638a84a1f07200309765/openedx/core/djangoapps/user_authn/views/utils.py#L85))

That means, it will not be available in response when `skip_registeration_form` is disabled and will be available if `skip_registeration_form` is enabled. 
How would this affect this field in serializer? I think this implementation means it will be available always in the MFE context? could you please explain that what will be the behaviour?

Reply:
It will always be available, but with False.